src/main.rs

```diff
@@ -289,12 +324,13 @@ fn service(config: Config) -> std::io::Result<()> {
             let gpu_temp = controller.hw_mon.gpu_temp().unwrap_or_default();
 
             let speed = config.speed_for_temp(gpu_temp);
-            if controller.hw_mon.fan_min() > speed || controller.hw_mon.fan_max() < speed {
+            let target_pwm = (speed as f32 * 2.55).round() as u32;
```

Comment:
NOTE: @Eraden 

I didn't noticed this during review. Documentation need to be updated